api/DOMStringList.json

```diff
@@ -148,7 +148,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMStringList/length",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "44"
```

Comment:
This is a tricky one. `DOMStringList` is legacy by now, but is used for `location.ancestorOrigins` and 3 different IndexedDB attributes, `objectStoreNames`, `indexNames`, and `objectStoreNames`.

I think some of those were available before Chrome 44, so I doubt this change. That this is close to Chrome 43 also makes me suspicious, see https://github.com/mdittmer/mdn-confluence/commit/5e173b18caedd5dd4aec38675712d514c9fe4784.

Can you dig around a bit?

Reply:
This seems to be another case just like `api.DOMException.code` up above -- property was around in the [first available commit for the interface](https://source.chromium.org/chromium/chromium/src/+/8d75ff802c1d2d66f5224851415b955343a18c42:third_party/WebKit/WebCore/dom/DOMCoreException.idl;bpv=0), but isn't defined in the prototype.  Hmmmm...